src/components/TextField/tests/TextField.test.tsx

```diff
@@ -379,6 +379,33 @@ describe('<TextField />', () => {
 
       expect(characterCount.text()).toBe('4/10');
     });
+
+    it('announces updated character count only when input field is in focus', () => {
+      const quietTextField = mountWithAppProvider(
+        <TextField
+          value="test"
+          showCharacterCount
+          label="TextField"
+          id="MyField"
+          onChange={noop}
+        />,
+      );
+
+      expect(quietTextField.find('#MyFieldCharacterCounter').prop<string>('aria-live')).toBe('off');
+
+      const politeTextField = mountWithAppProvider(
```

Comment:
Not sure if there's a more elegant way to force this component to re-render with updated props. The tests didn't update aria-live if I set the prop to focused or gave focus manually.

Reply:
We could split these into separate tests. For reference 
```tsx
quietTextField.setProps({...})
```
would cause a re-render with the new props 😄

Reply:
I dug a bit further. We're not updating `this.state.focus` when props update, only during the constructor. Using ` textField.find('input').simulate('focus');` fixed this, and better shows how this feature is supposed to work, too! 🎉